api/api_test.go

```diff
@@ -243,6 +243,14 @@ func TestMineGetBlockTemplate(t *testing.T) {
 			t.Fatal(err)
 		}
 
+		// make sure the target is correct
+		cs, err := c.ConsensusTipState()
+		if err != nil {
+			t.Fatal(err)
+		} else if target != cs.PoWTarget() {
+			t.Fatalf("expected target %v, got %v", cs.PoWTarget(), target)
```

Comment:
```suggestion
			t.Fatalf("expected target %v, got %v", target, cs.PoWTarget())
```

Reply:
Isn't the expected target the target we just computed? `cs.PoWTarget`? The one we got from the API is `target`.